pass/pass.go

```diff
@@ -1,3 +1,6 @@
+//go:build linux || darwin
```

Comment:
slightly on the fence on this one; I think the code in this repo works fine on any platform (if there's a pass binary to shell out to?)

Reply:
agree, switched to `unix`.

Reply:
actually `!windows` looks better.

Reply:
So for the `pass` helper; I guess the thing I was wondering here was that;

- the code in this repository (the helper itself) is basically platform-agnostic (including Windows)
- the "only" limitation we have for _testing_ is that we don't have a `pass` binary on Windows
- so, wondering if we should actually exclude it on Windows?
- (but indeed, we can't currently run the tests, as those depend on a `pass` binary)
- ^^ do we know if there's an implementation of the `pass` binary for windows?

Reply:
It looks like there's at least one implementation (although last release is really old; https://github.com/mbos/Pass4Win)

Reply:
Tried to install Pass4Win in ci but binary is not named `pass.exe` :persevere: https://github.com/docker/docker-credential-helpers/actions/runs/5110526809/jobs/9186513917#step:9:8

So disable tests for windows in the meantime. Let me know if you prefer a specific tag like `exlucde_pass_tests` that would be just set in CI.